src/components/createBlogPost/CreateBlogPost.jsx

```diff
@@ -0,0 +1,57 @@
+import "./CreateBlogPost.css"
+import { useForm } from "react-hook-form";
+import { useState } from "react";
+import {useNavigate} from "react-router-dom";
+
+function CreateBlogPost() {
+    const { register, handleSubmit, formState: { errors } } = useForm();
+    const [data, setData] = useState("");
+    const navigate = useNavigate();
+
+    const onSubmit= (formData) => {
+        const fomattedData = {
+            title: formData.title,
+            subtitle: formData.subtitle,
+            content: formData.content,
+            author: formData.author,
+            created_at: new Date().toISOString(),
+            readTime: Math.ceil(formData.content.split(" ").length / 100),
```

Comment:
Ik zie dat je geen gebruik hebt gemaakt van een helper functie, maar hier direct de math functie in verwerkt hebt. Dat ziet er ook prima uit.